config_single_task_distro.go

```diff
@@ -3,13 +3,19 @@ package evergreen
 import (
 	"context"
 
+	"github.com/evergreen-ci/utility"
 	"github.com/pkg/errors"
 	"go.mongodb.org/mongo-driver/bson"
 )
 
 type ProjectTasksPair struct {
 	ProjectID    string   `bson:"project_id" json:"project_id"`
 	AllowedTasks []string `bson:"allowed_tasks" json:"allowed_tasks"`
+	AllowedBVs   []string `bson:"allowed_bvs" json:"allowed_bvs"`
+}
+
+func (p *ProjectTasksPair) AllowAll() bool {
+	return utility.StringSliceContains(p.AllowedBVs, "all") || utility.StringSliceContains(p.AllowedTasks, "all")
```

Comment:
Since our update to Go 1.24, we could do instead `slices.Contains(p.AllowedBVs, "all")` instead of using our utility package (we have a ticket to replace all our usages so if you don't want to change that's okay)